site/content/docs/02-template-syntax/01-svelte-files.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: .svelte files
+title: Svelte file sections
```

Comment:
Maybe we should name it just `Components`? It's briefer, but considering other frameworks' docs, I am conditioned to clicking on `Components` without even thinking, knowing it would have the component declration syntax in it

Reply:
yeah, I agree we could make it shorter. How about "Svelte components"?